src/utils/cluster_setup.py

```diff
@@ -0,0 +1,40 @@
+import click
+import subprocess
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+def cluster_setup(ctx: click.Context):
+    # Copy current environment
+    env = os.environ.copy()
+    commands = [
+        [
+            "oc",
+            "apply",
+            "--server-side",
+            "-k",
+            "https://github.com/kubeflow/trainer.git/manifests/overlays/runtimes?ref=master",
+        ],
+        [
+            "oc",
+            "apply",
+            "--server-side",
+            "-k",
+            "https://github.com/kubeflow/trainer.git/manifests/overlays/runtimes?ref=master",
+        ],
```

Comment:
I don't think these are needed.

Reply:
I think they are on a non-RHOAI cluster according to: https://www.kubeflow.org/docs/components/trainer/operator-guides/installation/

but we can remove them if we are not targeting that.

Reply:
These are overlays for the Trainer v2. We are still using v1 PyTorchJob API.

Reply:
ahhh ok, I will remove both of these 👍